### PR TITLE
Replace leftover fields with values

### PR DESCRIPTION
**Pull request by elsh**

### CODE REVIEW

File: swift/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.DS_Store
```
Comment (mkruber) [inline swift/.gitignore]: Some of this is already ignored at the root level `.gitignore`.

File: swift/Package.swift [hunk 2]
```diff
@@ -1,19 +1,3 @@
-/**
- *    Copyright (c) 2019 Uber Technologies, Inc.
```
Comment (elsh) [inline swift/Package.swift]: this confuses parsing toolchain version when running swift build

File: swift/tests/control.swift [hunk 3]
```diff
@@ -201,7 +202,7 @@ class SwiftExamples {
     }
 
     // tests for task T2191251
```
Comment (mkruber) [inline swift/tests/control.swift]: This comment about task name can be removed.

File: swift/Package.swift (hunk 2 above)
Comment (mkruber) [reply swift/Package.swift]: Since this is a generic file, not having the license should be okay.

**Comment by ketkarameya** [comment]
Since there has been no action on this PR for the past 2 years I am closing it.

**Pull request closed without merging**